### PR TITLE
fix(examples): update examples with required parameter destination

### PR DESCRIPTION
**Pull request by vdice**
Updates example bundle.json files in `examples` to specify the required parameter destination where applicable.

### CODE REVIEW

**Comment by vdice** [comment]
@technosophos I just realized that there are some doc files (`101-bundle-json.md`, etc.) that reference the example bundle.json contents.  A few questions:

1. It appears as of today it is a manual effort to re-copy updated examples into the appropriate locations of the docs?
2. I see some instances of corresponding canonical json representations, which I'm assuming should also be updated -- what tool was used to generate these representations?
3. Thoughts on how we might avoid the manual effort of 1 & 2 (if I have those right)?

**Comment by technosophos** [comment]
1. It is currently a manual effort. 🐼 
2. I left a note somewhere in 101 saying `Small snippets of JSON may be shown in the order in which the fields are described (for clarity) rather than in Canonical JSON order.` -- mainly because I didn't think I could catch and correct all of the fields. Ideally, though, they should be in canonical JSON
3. I guess we could use a preprocessor to insert them. It would probably be good to see how other places do this.

**Comment by vdice** [comment]
Last commit updated the json snippets in corresponding docs; PTAL @technosophos

File: 101-bundle-json.md [hunk 1]
```diff
@@ -88,67 +91,70 @@ Source: [101.01-bundle.json](examples/101.01-bundle.json)
...
-        "digest": "sha256:bbbbbbbbbbbb...",
-        "image": "technosophos/helloworld:0.1.2",
-        "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
-        "platform": {
-          "architecture": "amd64",
-          "os": "linux"
-        },
-        "size": 1337
-      }
+  "credentials": {
```
Comment (vdice) [inline 101-bundle-json.md]: indentation fix here... looked like original had bulk of json body outdented by 2 spaces.

**Pull request merged**